js/background/background.js

```diff
@@ -677,6 +677,8 @@ let actionCallbacks = new Map([
     ['inventory.coupons', SteamCommunity.coupons], // #3
     ['inventory.gifts', SteamCommunity.gifts], // #1
     ['inventory.community', SteamCommunity.items], // #6
+
+    ['error.test', () => { return Promise.reject(new Error("This is a TEST Error. Please ignore.")); }],
```

Comment:
For what reason did you introduce this here?

Reply:
It's a test error, it's not called in ordinary operation but I was finding it useful to `Background.action('error.test')` from the Chrome dev console to verify the new logging worked.